build breakage - linux 6.6 - no cfg80211_ap_update symbol definition

I am using armbian build of linux 6.6 and get:
```
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c:4202:24: warning: 'struct cfg80211_ap_update' declared inside parameter list will not be visible outside of this definition or declaration  
[🐳|🔨]    4202 |                 struct cfg80211_ap_update *params)                                                                                                                                               
[🐳|🔨]         |                        ^~~~~~~~~~~~~~~~~~                                                                                                                                                        
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c: In function 'cfg80211_rtw_change_beacon':                                                                                                  
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c:4204:60: error: invalid use of undefined type 'struct cfg80211_ap_update'                                                                   
[🐳|🔨]    4204 |                 struct cfg80211_beacon_data *info = &params->beacon;                                                                                                                             
[🐳|🔨]         |                                                            ^~                                                                                                                                    
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c: At top level:                                                                                                                              
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c:6861:26: error: initialization of 'int (*)(struct wiphy *, struct net_device *, struct cfg80211_beacon_data *)' from incompatible pointer ty
pe 'int (*)(struct wiphy *, struct net_device *, struct cfg80211_ap_update *)' [-Werror=incompatible-pointer-types]                                                                                                
[🐳|🔨]    6861 |         .change_beacon = cfg80211_rtw_change_beacon,                                                                                                                                             
[🐳|🔨]         |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                              
[🐳|🔨]   drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.c:6861:26: note: (near initialization for 'rtw_cfg80211_ops.change_beacon')                                                                   
[🐳|🔨]   cc1: some warnings being treated as errors                                                                                                                                                               
[🐳|🔨]   make[7]: *** [scripts/Makefile.build:243: drivers/net/wireless/rtl8723ds/os_dep/linux/ioctl_cfg80211.o] Error 1                                                                                          
[🐳|🔨]   make[6]: *** [scripts/Makefile.build:480: drivers/net/wireless/rtl8723ds] Error 2              
[🐳|🔨]   make[6]: *** Waiting for unfinished jobs....               
```

this seems to relate to latest commit https://github.com/lwfinger/rtl8723ds/commit/31100220ebc66d4c2bb023bbb2ab751269d84480 in this repository from pull request 43.
Was it build-tested?
I cannot find any reference to cfg80211_ap_update in https://elixir.bootlin.com/linux/latest/A/ident/cfg80211_ap_update


I see no change in the signature for change_beacon  https://elixir.bootlin.com/linux/latest/source/include/net/cfg80211.h#L4452

--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -4188,7 +4188,7 @@ static int cfg80211_rtw_start_ap(struct wiphy *wiphy, struct net_device *ndev,
 	return ret;
 }
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 5, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 7, 0))
 static int cfg80211_rtw_change_beacon(struct wiphy *wiphy, struct net_device *ndev,
 		struct cfg80211_ap_update *params)
 {

